treeherder/config/settings.py

```diff
@@ -27,6 +27,9 @@ def server_supports_tls(url):
 
 # Default to retaining data for ~4 months.
 DATA_CYCLE_DAYS = env.int("DATA_CYCLE_DAYS", default=120)
+# Determines the number of jobs we try to delete per iteration when
+# cycling data
+DATA_CYCLE_CHUNK_SIZE = env.int("DATA_CYCLE_CHUNK_SIZE", default=1000)
```

Comment:
Later update: 250 still causes OOMs, though it takes a while. But 100 seems to succeed, and stage stays responsive. Should probably stick with that.